Lib/_android_support.py

```diff
@@ -31,16 +31,19 @@ def init_streams(android_log_write, stdout_prio, stderr_prio):
     logcat = Logcat(android_log_write)
 
     sys.stdout = TextLogStream(
-        stdout_prio, "python.stdout", sys.stdout.fileno(),
-        errors=sys.stdout.errors)
+        stdout_prio, "python.stdout", sys.stdout.fileno())
     sys.stderr = TextLogStream(
-        stderr_prio, "python.stderr", sys.stderr.fileno(),
-        errors=sys.stderr.errors)
+        stderr_prio, "python.stderr", sys.stderr.fileno())
```

Comment:
Not sure I understand what is going on here - why is the `errors` kwarg no longer required?

Reply:
Because it's been added as a default in TextLogStream itself.

Reply:
The idea is that because of how the output is being logged, we want to explicitly use backslashreplace, rather than copying the existing values.

Reply:
Got it - I was misreading what `sys.stdout.errors` was passing in.